sdk/core/abort-controller/src/AbortController.ts

```diff
@@ -58,8 +58,34 @@ export class AbortError extends Error {
 export class AbortController {
   private _signal: AbortSignal;
 
-  constructor() {
+  /**
+   * @param {AbortSignal} [parentSignal] The AbortSignal that will signal aborted on the AbortSignal associated with this controller.
+   * @constructor
+   */
+  constructor(parentSignals?: AbortSignalLike[]);
+  constructor(...parentSignals: AbortSignalLike[]);
```

Comment:
Every overload should get its own jsdocs right?

Reply:
Yes, missed updating this JSDOC, thanks for catching!